command/sub8_launch/launch/subsystems/perception.launch

```diff
@@ -1,18 +1,26 @@
 <launch>
-  <!-- Try this! -->
-  <!-- roslaunch sub8_launch perception.launch color_calibration:=`rospack find sub8_launch`/config/gazebo_color_calibration.yaml -->
+  <!-- roslaunch sub8_launch perception.launch color_calibration:=gazebo_color_calibration.yaml -->
   <arg name="color_calibration" default="default_color_calibration.yaml" />
 
   <rosparam file="$(find sub8_launch)/config/course_geometry.yaml" />
-  <!-- <rosparam file="$(find sub8_launch)/config/default_color_calibration.yaml" /> -->
   <rosparam file="$(find sub8_launch)/config/$(arg color_calibration)" />
 
-  <node pkg="sub8_perception" type="follow_orange_pipes.py" name="sub_orange_pipes" output="screen" />
-  <node pkg="sub8_perception" type="buoy_2d.py" name="buoy_2d" output="screen" />
+  <node pkg="sub8_perception" type="buoy_2d.py" name="buoys" output="screen">
+    <rosparam param="use_boost">True</rosparam>
```

Comment:
It could be cleaner if we just create config files for our nodes and import them here so these launch files don't get so cluttered. I'm doing that for torpedo board stuff although I havent pushed it yet. Think about it for a future PR.